static/styles/subscriptions.css

```diff
@@ -222,7 +222,7 @@
 
     .subscriber_list_container {
         position: relative;
-        max-height: 300px;
+        max-height: 440px;
```

Comment:
How is this magic number determined?  We should use a `calc` or SCSS variable to connect it with whatever other numbers it needs to equal.

Reply:
This suggests the value only works for your screen size:

![image](https://user-images.githubusercontent.com/2746074/124837192-39283b80-df39-11eb-972c-82c11b915c43.png)

Reply:
As a sidenote, the scrollbar overlaps the "unsusbcribe" buttons; we need a bit more right margin on this widget.

Reply:
I changed its `max-height` to `100%`. So that it takes full available space.